### ISSUE TITLE
generalSettings no longer contains "url" on multisite

### ISSUE THREAD

**Issue by reallyfabdigital** [wp-graphql/wp-graphql#1334]
The "url" attribute is no longer returned from generalSettings on a multisite WordPress site. Is there a way to get it back in please?

Thank you.

**Comment by HumidBrains** [comment]
This bug unfortunately makes it impossible to use gatsby-source-wordpress-experimental as described here https://github.com/gatsbyjs/gatsby-source-wordpress-experimental/issues/23

**Comment by riddla** [comment]
I'm in the same boat, hitting this when trying to move from `gatsby-source-wordpress` to `gatsby-source-wordpress-experimental`. Would love to help if you point me to a direction in the code ...

**Comment by jasonbahl** [comment]
Hey! 

Sorry for the inactivity here. This is addressed in the v0.10.0 release I'm about to publish. (https://github.com/wp-graphql/wp-graphql/pull/1348)

It's a big release, so if you need to get this working but don't have time to update, you should be able to do the following as a stop-gap:

```
add_action( 'graphql_register_types', function() {
  if ( is_multisite() ) {
    register_graphql_field( 'GeneralSettings', 'url', [
      'type' => 'String',
      'resolve' => function() {
          return get_site_url();
      }
    ] );
  }
} );

**Comment by jasonbahl** [comment]
You can see this in action here, querying from the main site and a subdirectory multisite site: 

![Screen Shot 2020-06-26 at 1 16 25 PM](https://user-images.githubusercontent.com/1260765/85894307-b90db880-b7b1-11ea-925b-334dffeb1c85.png)
![Screen Shot 2020-06-26 at 1 16 37 PM](https://user-images.githubusercontent.com/1260765/85894310-b9a64f00-b7b1-11ea-8a6a-186ffe8b18f2.png)
